src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -269,7 +269,7 @@ const useGroupInfo = (channelId = null) => {
 
     try {
       const {channelName} = getChannelListInfo(
-        responseChannelData.data,
+        responseChannelData.data?.channel,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is consistent with the previous one and maintains the same level of null safety.
```diff
- responseChannelData.data,
+ responseChannelData.data?.channel,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->